getting-started/setup-multi-node-advanced.md

```diff
@@ -0,0 +1,370 @@
+To reload the server configuration, use the following command:
+```bash
+pg_ctl reload
+```
+
+##### 4. Add the user to the data node
+Within the PostgreSQL instance(psql) issue the command:
+
+```sql
+CREATE ROLE new_user WITH LOGIN PASSWORD 'xyzzy';
+```
+
+### Authentication on the access node
+To execute queries across data nodes, the access node needs to connect
+to data nodes as the user executing the query. This means that the
+access node needs to be configured for password authentication and
+needs the passwords for the relevant user on each data node.
+
+##### 5. Configure password encryption method
+This step is identical to (1) except applied to the access node.  Find the `postgresql.conf` file and set the `password_encryption` parameter:
+```
+password_encryption = 'scram-sha-256'
+```
+
+##### 6. Create a password file
+The password file enables the access node to connect securely to data 
+nodes and is by default located in the data directory ([PostgreSQL documentation][passfile]). If a file doesn't exist, create one with the 
+new user information.  Ex: from the terminal:
+
+```bash
+cd <datadir>
+cat >passfile <<EOT
+*:*:*:new_user:xyzzy
+EOT
+```
+Then apply the correct permissions to the file:
+```
+cd <datadir>
+chmod 0600 passfile
+```
+
+##### 7. Reload server configuration
+Reload the server configuration with:
+```
+pg_ctl reload
+```
+and the system setup is complete
+
+---
+
+## Certificate authentication [](multi-node-auth-certificate)
+This method is more complex to set up than password authentication, but 
+more secure and easier to automate.
+
+The steps to set up certificate authentication are:
+
+1. Set up a certificate authority that can sign certificates and
+   create a root certificate.
+2. Generate a key and certificate for the access node and data nodes.
+3. Configure the access node and data node to use the certificates and
+   to accept certified connections.
+4. Generate a key and certificate for each user.
+
+To use certificates, each instance involved in certificate
+authentication uses three files:
+
+- A *root certificate*, which we assumed to be named `root.crt`. This is 
+  the certificate that identifies a trusted Certification Authority (CA). 
+  It contains the public key of the CA (or CAs, in the event that there 
+  is a certificiation authority chain) and can be freely copied between 
+  servers.
+- A *certificate* that is signed by the CA. This contains the public
+  key and is used to verify signatures and can also be freely copied.
+- A *key* that contains the private key. This is used to sign messages
+  and should be kept secure on the instance where it is generated.
+
+For the access node a single *root certificate* can be shared among 
+users. However each user on the access node needs a unique *signed 
+certificate* and *key*.
+
+For each data node, the *root certificate* can be copied from the access 
+node, but a *signed certificate* and *key* must be generated.
```

Comment:
Again, I am not sure what this is trying to say. It sounds as if it is repeating what was already said above in the bullet list. I suggest removing.